Data/Conduit/Find.hs

```diff
@@ -460,12 +461,12 @@ glob g = case parseOnly globParser (pack g) of
                 <*> char ']'
         <|> do
             x <- anyChar
-            return . pack $ if x `elem` ".()^$"
+            return . pack $ if x `elem` (".()^$" :: String)
```

Comment:
This is for ghc 7.10.
See https://ghc.haskell.org/trac/ghc/wiki/Migration/7.10.